README.md

```diff
@@ -55,6 +62,7 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
      * [End-To-End SQL Example](#end-to-end-sql-example)
      * [Configuring the SQL Database to Match the GraphQL Schema](#configuring-the-sql-database-to-match-the-graphql-schema)
   * [Miscellaneous](#miscellaneous)
+     * [Pretty Printing GraphQL Queries](#pretty-printing-graphql-queries)
```

Comment:
Let's hyphenate `Pretty-Printing` since it should be one word (see line 1316 below, for example).

Reply:
I used the online editor to address this myself.